secator/hooks/json.py

```
@@ -0,0 +1,157 @@
+						try:
+							items.append(json.loads(line))
+						except json.JSONDecodeError as e:
+							debug(f'Bad JSON in {sidecar_path}: {e}', sub='hooks.json')
+			except Exception as e:
+				debug(f'Error reading sidecar {sidecar_path}: {e}', sub='hooks.json')
+		results[type_name] = items
+	try:
+		with open(report_path, 'w') as f:
+			json.dump({'info': info, 'results': results}, f, cls=DataclassEncoder, indent=2)
+		debug(f'Wrote final JSON report to {report_path}', sub='hooks.json')
+	except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)